gateway-home/modules/ROOT/pages/gateway-overview.adoc

```diff
@@ -6,167 +6,48 @@ endif::[]
 
 An API gateway is an intermediary between a client application and a set of microservices or backend APIs. Responsibilities include request routing, rate limiting, caching, authentication, authorization, security, monitoring, and logging.
 
-The gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns.
+A gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns.
```

Comment:
I prefer "enables" to "allows" unless permission is involved.

Reply:
Fixed

Reply:
still says allows :)